### PR TITLE
WEBEU2 - UI III Flexbox - Stretch Wasiu Idowu 🇳🇬 

### PR DESCRIPTION
**Pull request by Hoxtygen**
The essence of this PR is to meet the stretch goals  stated in the project

### CODE REVIEW

File: great-idea/products.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (maximesalomon) [inline great-idea/products.html:2]: Great job o the products page :)

File: great-idea/services.html [hunk 2]
```diff
@@ -1,10 +1,11 @@
 <!doctype html>
 
 <html lang="en">
```
Comment (maximesalomon) [inline great-idea/services.html:3]: Great job as well for the services page!

File: great-idea/services.html [hunk 3]
```diff
@@ -15,93 +16,117 @@
...
+    <section class="company-business">
+      <div class="services">
+        <h3>Services</h3>
+        <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus
+          rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur
+          cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
+          facilisis iaculis.</p>
+      </div>
+      <div class="business-item-container">
+        <article class="company-business-item">
```
Comment (maximesalomon) [inline great-idea/services.html:57]: `<article>` should only be used for specific content like news article or comments - https://www.w3schools.com/tags/tag_article.asp

**Review by maximesalomon — COMMENTED** [review]
Absolutely fantastic job on the assignment Wasiu! You successfully implemented all of the tasks and have completed every single one of the stretch goals!

Overall great job 🚀

File: great-idea/services.html (hunk 3 above)
Comment (Hoxtygen) [reply great-idea/services.html:57]: Okay. Thanks for the feedback.

**Pull request merged**